### PR TITLE
Appbar view

### PR DESCRIPTION
**Pull request by jonzee**

### CODE REVIEW

File: src/sass/AppBar.scss [hunk 1]
```diff
@@ -1,11 +1,45 @@
...
-  position: fixed;
-  z-index: 1100;
-  width: 100%;
-  margin: 64px 0 0 -24px;
-  height: 1px;
-  background: #B5B5B5;
-  transform: translate3d(0,0,0);
-  -moz-transform: translate3d(0,0,0);
-  -webkit-transform: translate3d(0,0,0);
+.app-bar {
```
Comment (michalmikolajczyk) [inline src/sass/AppBar.scss]: not a fan of the `!important`

have you tried instead:
`.app-bar.app-bar {
  // ...
```

File: src/ts/Router/index.tsx [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
+import NotFound from '../NotFound'
 
 export default function AppRouter(props) {
   return (
     <Router history={browserHistory}>
       <Route path="/" component={Offers} />
       <Route path="/detail" component={Detail} />
+      <Route path="/signup" component={NotFound} />
+      <Route path="/login" component={NotFound} />
+      <Route path="/bike_owners" component={NotFound} />
```
Comment (michalmikolajczyk) [inline src/ts/Router/index.tsx]: let's go with `bike-owners`
https://support.google.com/webmasters/answer/76329?hl=en

File: src/ts/Search/Location.tsx [hunk 3]
```diff
@@ -27,18 +27,25 @@ export default class Location extends React.Component<any, any> {
 
   componentDidMount() {
     // in case of re-creating component we do not need another script
-    if (document.getElementById('google-maps-script')) {
-      this.initMap();
-      return;
+    if (typeof google !== 'undefined') {
```
Comment (michalmikolajczyk) [inline src/ts/Search/Location.tsx]: why not
```
if (google) return this.initMap()
```
?

File: src/sass/AppBar.scss (hunk 1 above)
Comment (jonzee) [inline src/sass/AppBar.scss]: The problem here is that I'm overwriting inline style of `material-ui` component so there are two way: add inline styling in `javascript` or add styles with `!important`

File: src/ts/Search/Location.tsx (hunk 3 above)
Comment (jonzee) [inline src/ts/Search/Location.tsx]: Perhaps it's a slight exaggeration but http://stackoverflow.com/questions/3390396/how-to-check-for-undefined-in-javascript

File: src/sass/AppBar.scss (hunk 1 above)
Comment (michalmikolajczyk) [inline src/sass/AppBar.scss]: jasne. Przy okazji douczyłem się https://stuffandnonsense.co.uk/archives/css_specificity_wars.html
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity

File: src/ts/Search/Location.tsx (hunk 3 above)
Comment (michalmikolajczyk) [inline src/ts/Search/Location.tsx]: in this case, I think the variable is declared, so no reference error here. Alas, do not follow my initial comment! Looking at the link you provided, the `typeof` strategy seems to be a good practice.

**Pull request merged**